packages/schematics/angular/application/files/src/environments/environment.ts

```diff
@@ -1,7 +1,7 @@
 // The file contents for the current environment will overwrite these during build.
 // The build system defaults to the dev environment which uses `environment.ts`, but if you do
 // `ng build --env=prod` then `environment.prod.ts` will be used instead.
```

Comment:
Can you change the last two lines to the following please? A lot of what's written here is not applicable anymore.
```
// This file can be replaced during build by using the `fileReplacements` array.
// `ng build ---prod` replaces `environment.ts` with `environment.prod.ts`.
// The list of file replacements can be found in `angular.json`.
``

Reply:
Sure

Reply:
Actually, all 4 lines should change to that. Sorry, I was rewording it Midway.

Reply:
Ok, let me know if that is correct